src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -0,0 +1,644 @@
+these optimizations.
+
+### Use server-side rendering
+
+Minimizing the amount of JavaScript should always be the first thing to focus on for sites that are
+mostly client-rendered. However, you should also consider combining a server rendering experience to
+improve LCP as much as possible. 
+
+This concept works by using the server to render the application into HTML, where the client then
+"[hydrates](https://www.gatsbyjs.org/docs/react-hydration/)" all the JavaScript and required data
+onto the same DOM content. This can improve LCP by ensuring the main content of the page is first
+rendered on the server rather than only on the client, but there are a few drawbacks:
+
++   Maintaining the same JavaScript-rendered application on the server and the client can
+    increase complexity.
++   Executing JavaScript to render an HTML file on the server will always increase server
+    response times (TTFB) as compared to just serving static pages from the server.
++   A server-rendered page may look like it can be interacted with, but it can't respond to any
+    user input until all the client-side JavaScript has executed. In short, it can make [**Time to
+    Interactive**](/tti/) (TTI) worse.
+
+### Use pre-rendering
+
+Pre-rendering is a separate technique that is less complex than server-side rendering and also
+provides a way to improve LCP in your application. A headless browser, which is a browser without a
+user interface, is used to generate static HTML files of every route during build time. These files
+can then be shipped along with the JavaScript bundles that are needed for the application.
+
+With pre-rendering, TTI is still negatively impacted but server response times aren't as affected as
+they would be with a server-side rendering solution that dynamically renders each page only after
+it's requested.
+
+<figure class="w-figure">
+  <img src="./lcp-prerendering.png" alt="Example of LCP improvement: Before and after pre-rendering">
+  <figcaption class="w-figcaption">Example of LCP improvement: Before and after pre-rendering</figcaption>
+</figure>
+
+{% Aside %}
+For a deeper dive into different server-rendering architectures, take a look at [Rendering on the
+web](https://developers.google.com/web/updates/2019/02/rendering-on-the-web).
+{% endAside %}
+
+## Developer tools
+
+A number of tools are available to measure and debug Largest Contentful Paint (LCP).
+
++   [Lighthouse 6.0](https://developers.google.com/web/tools/lighthouse) includes support for
+    measuring LCP in a lab setting.
+
+    ![Lighthouse 6.0](./lighthouse.jpg)
+
++   The **Timings** section of the
+    [Performance](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance)
+    panel in Chrome DevTools includes a LCP marker and shows you which element is associated with
+    LCP when you hover over the **Related Node** field.
+
+    ![LCP in Chrome DevTools](./lcp-chrome-devtools.png)
```

Comment:
This image is OK because the text that's essentially an ad for a Google product is pretty small and not noticeable.

Reply:
Updated to design.google here as well :)